RELEASE.md

```diff
@@ -12,58 +20,55 @@
   - Major releases: `bumpversion --commit --tag --new-version 1.0.0 major`
 4. Deploy to pypi
   - `python setup.py sdist upload -r pypi`
-5. Deploy to homebrew
-  - Make a pull request against homebrew-core
-6. Deploy to conda-forge
-  - Make a pull request against dbt-feedstock
+5. Deploy to homebrew (see below)
+6. Deploy to conda-forge (see below)
 7. Git release notes (points to changelog)
 8. Post to slack (point to changelog)
 
+After releasing a new version, it's important to merge the changes back into the other outstanding release branches. This avoids merge conflicts moving forward.
```

Comment:
I think ideally, higher-version release branches would be periodically pulling in lower-version release branches during the process, rather than waiting for release time. I'm mostly imagining long-lived feature branches here.

Reply:
yup, that's a good point, these branches will probably diverge enough that cherry-picking commits instead of merging is a good idea.